tests/python/unittest/test_onnx.py

```diff
@@ -0,0 +1,95 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import os
+import pytest
+import tempfile
+
+import mxnet as mx
+from mxnet import nd, sym
+from mxnet.gluon import nn
+from mxnet.contrib import onnx as onnx_mxnet
+
+
+def _assert_sym_equal(lhs, rhs):
```

Comment:
That's where they are! I searched for onnx under unit-test and didn't find anything.

Reply:
this file is in `tests/python/unittest`
the onnx tests are under `tests/python-pytest/onnx`

The reason we did not put onnx tests under unittest folder is because onnx tests uses pytest framework for testing, while the rest of mxnet uses nosetests.